requirements/docs.txt

```diff
@@ -20,3 +20,5 @@ myst-parser
 ipywidgets
 plotly>=4.14.0
 kaleido
+panel==0.12.4
+holoviews==1.14.6
```

Comment:
Can you please relax the dependency versions so that they are just lower bounds, instead of exact matches?